org-formation/organization.yaml

```diff
@@ -261,6 +262,16 @@ Organization:
         Department: Platform
         Project: Infrastructure
 
+  NlpSandboxAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: NlpSandboxAccount
+      RootEmail: aws-nlpsandbox@sagebase.org
+      Alias: org-sagebase-nlpsandbox
+      Tags:
+        Department: CompOnc
+        Project: NLP-Challenge
```

Comment:
you may want to add the buget-alarm-threshold tags, similar to what's on the  `MobileHealthDataEngineeringAccount`.  That will set a max budget for this account and if the costs breaches the max it will send a notification to the email-recipient.